src/Installer/ExtensionInstaller.php

```diff
@@ -52,15 +52,13 @@ public function __construct(
     }
 
     /**
-     * Decides if the installer supports the given type
-     *
      * @param  string $packageType
      * @return bool
      */
     public function supports($packageType)
     {
-        return $packageType !== 'typo3-cms-core'
-            && strncmp('typo3-cms-', $packageType, 10) === 0;
+        return $packageType === 'typo3-cms-extension'
+            || $packageType === 'typo3-cms-framework';
```

Comment:
please create a ticket that describes a scenario, the current behaviour and your expected behaviour

Reply:
In any case: this is a breaking change, which requires a new major version